docs/_ext/codesnippet.py

```diff
@@ -79,7 +79,7 @@ def _set_options(self) -> None:
             "start-after", self.default_start_marker
         )
         self.options["end-before"] = self.options.get(
-            "end-after", self.default_end_marker
+            "end-before", self.default_end_marker
```

Comment:
Was it a bug?

Reply:
YES 👁️ 👄 👁️

Reply:
Yes, I don't know how the fact that `literalinclude` had to be used instead of `codesnippet` didn't make this bug resolved earlier lol

Reply:
I just thought that you didn't implement that feature in the `codesnippet` 😅